File: 3. Sophomore_HKI/OOP/OOP_CUOIKI/QuanLyShopQuanAo (C#)/NoiCungCap.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyShopQuanAo
{
    public class NoiCungCap
    {
        public string maNoiCungCap { get; set; }     // primary key
        public string tenNoiCungCap { get; set; }
        public string diaChi { get; set; }
        public string SDT { get; set; }

        // constructor
        public NoiCungCap() { }
        public NoiCungCap(NoiCungCap noiCungCap)
        {
            this.maNoiCungCap = noiCungCap.maNoiCungCap;
            this.tenNoiCungCap = noiCungCap.tenNoiCungCap;
            this.diaChi = noiCungCap.diaChi;
            this.SDT = noiCungCap.diaChi;
        }
        public NoiCungCap(string maNoiCungCap, string tenNoiCungCap, string diaChi, string SDT)
        {
            this.maNoiCungCap = maNoiCungCap;
            this.tenNoiCungCap = tenNoiCungCap;
            this.diaChi = diaChi;
            this.SDT = SDT;
        }           
    }
}

File: 3. Sophomore_HKI/OOP/OOP_CUOIKI/QuanLyShopQuanAo (C#)/src/mypack/NoiCungCap.java
package mypack;

public class NoiCungCap {
	
	public String maNoiCungCap;		// primary key
    public String tenNoiCungCap; 
    public String diaChi;
    public String SDT;

    // constructor
    public NoiCungCap() { }
    public NoiCungCap(NoiCungCap ncc)
    {
        this.maNoiCungCap = ncc.maNoiCungCap;
        this.tenNoiCungCap = ncc.tenNoiCungCap;
        this.diaChi = ncc.diaChi;
        this.SDT = ncc.diaChi;
    }
    public NoiCungCap(String maNoiCungCap, String tenNoiCungCap, String diaChi, String SDT)
    {
        this.maNoiCungCap = maNoiCungCap;
        this.tenNoiCungCap = tenNoiCungCap;
        this.diaChi = diaChi;
        this.SDT = diaChi;
    }      
    
    // getter - setter
    public String getMaNoiCungCap() {
    	return maNoiCungCap;
    }
    public void setMaNoiCungCap(String maNoiCungCap) {
    	this.maNoiCungCap = maNoiCungCap;
    }
    public String getTenNoiCungCap() {
		return tenNoiCungCap;
	}
	public void setTenNoiCungCap(String tenNoiCungCap) {
		this.tenNoiCungCap = tenNoiCungCap;
	}
	public String getDiaChi() {
		return diaChi;
	}
	public void setDiaChi(String diaChi) {
		this.diaChi = diaChi;
	}
	public String getSDT () {
		return SDT;
	}
	public void setSDT(String SDT) {
		this.SDT = SDT;
	}
}
